ui/client/stylesheets/processes.styl

```diff
@@ -206,4 +206,7 @@
   margin auto
 
 .transparent
-  background-color: rgba(0, 0, 0, 0)
\ No newline at end of file
+  background-color: rgba(0, 0, 0, 0)
+
+.validation-label
+  color red
```

Comment:
why not use variable we have for color of warnings?

Reply:
Good point, done